README.md

```diff
@@ -166,8 +166,8 @@ The following sensors are calculated using values from the weather station. Note
 | Name              | Description                                                    | Ambient Weather | Weather Underground | Unit      |
 | ----------------- | -------------------------------------------------------------- | --------------- | ------------------- | --------- |
 | dewpoint          | Outdoor dewpoint temperature                                   | Yes             | No                  | °F        |
-| dewpoint1..10     | Outdoor dewpoint temperature for sensors 1 through 10          | Yes             |                     | feelsLike | Windchill, heat index, or temperature | Yes | Yes | F |
+| dewpoint1..10     | Outdoor dewpoint temperature for sensors 1 through 10          | Yes             | Yes                 | °F        |
+| feelsLike         | Windchill, heat index, or temperature                          | Yes             | Yes                 | F         |
```

Comment:
Minor, but for consistency's sake. This and the following line should have a degrees symbol like the others. So `°F`

Reply:
Fixed